react-native/ReactNativeFlipperExample/ios/Podfile

```diff
@@ -70,12 +70,8 @@ target 'ReactNativeFlipperExample' do
   pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
-  target 'ReactNativeFlipperExampleTests' do
-    inherit! :complete
-    # Pods for testing
-  end
-
-  use_native_modules!
+  # use_native_modules!
+  pod 'react-native-flipper', :path => "../../react-native-flipper", :configuration => 'Debug'
```

Comment:
How about not disabling autolinking and use configure CLI to see local lib? https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#how-can-i-autolink-a-local-library

Reply:
note that relative-deps has been set up in this example project, so running `yarn relative-deps` in the root should already copy the module to make it local (assuming that works for iOS as well, it works fine for Android)

Reply:
Hm, what I mean is that we don't need to disable autolinking (commenting out the `use_native_modules!`). With the right config, we could also get rid of the `pod` declaration, but that's fine and actually less code :D

Reply:
Yea the example app setup changes were temporary just so I can get it building. The problem I had is I wanted to open the xcodeworkspace for the example app and edit files from there but using autolinking + relativedeps it would use a copy under node_modules instead. Or at least thats what I thought it did.